modules/ztp-creating-siteconfig-custom-resources.adoc

```diff
@@ -71,8 +71,7 @@ spec:
    releaseImage: quay.io/openshift-release-dev/ocp-release:4.10.0-x86_64 <2>
 ----
 <1> The descriptive version that you want to deploy.
-<2> Points to the specific release image to deploy.
-
+<2> Specifies the `releaseImage` to deploy and determines the OS Image version. The discovery ISO is based on an OS image version as the specified `releaseImage`, or latest if the exact version is unavailable.
```

Comment:
maybe a small nit but is the second "specified" needed. "The discovery ISO is based on an OS image version as the  `releaseImage`, or latest if the exact version is unavailable."

Reply:
Updated as suggested.